src/config/crd/bases/goharbor.goharbor.io_inspectionpolicies.yaml

```diff
@@ -155,8 +155,8 @@ spec:
                       governor:
                         description: Indicate whether to config of governor
                         properties:
-                          apiToken:
-                            description: Api token for user authentication
+                          cspSecretName:
+                            description: Secret name where CSP api token is stored
```

Comment:
Change it to "Secret name where CSP api token is stored in cnsi-system namespace